CHANGELOG.md

```diff
@@ -42,6 +42,20 @@ impersonation attacks) by default.
 
 To restore the previous behaviour, set `ssl.endpoint.identification.algorithm` to `none`.
 
+## Known Issues
+
+### Consumer issues
+
+ * The Consumer Batch APIs `rd_kafka_consume_batch` and `rd_kafka_consume_batch_queue`
+   are not thread safe if `rkmessages_size` is greater than 1 and any of the **seek**,
+   **pause**, **resume** and **repartition** operation is performed in parallel. Some
+   of the messages might be lost in the above scenario.
```

Comment:
might be lost, or erroneously returned to the application, in the above scenario.